nest_elephant_tvb/translation/FatEndPoint.py

```diff
@@ -0,0 +1,196 @@
+from mpi4py import MPI
+import pathlib
+
+def make_connections(path_to_files_receive, path_to_files_send, logger_master):
+    '''
+    Fat end point, here: connections to nest and tvb.
+    '''
+    use_mpi = True
+    # init MPI
+    if use_mpi:
```

Comment:
If statement does not have any function: use_mpi = True in line 8

Reply:
I will leave it, this will obviously change as soon as the End Point is properly implemented. Does not hurt either at the moment.